src/main/java/redis/clients/jedis/args/StreamEntryID.java

```diff
@@ -0,0 +1,8 @@
+package redis.clients.jedis.args;
```

Comment:
@sazzad16 what about the original StreamEntryID class?

Reply:
@gkorland Let's discuss...